lib/foreman_chef/engine.rb

```diff
@@ -33,6 +33,10 @@ class Engine < ::Rails::Engine
       ActionView::Base.send :include, ChefProxyForm
     end
 
+    initializer 'foreman_chef.dynflow_world' do |app|
+       ForemanTasks.dynflow.require!
```

Comment:
I have a WIP version from a month ago lying around here, only the init script would have to be rewritten, I'm adding this once again on my TODO list...